src/daemon/rpmostreed-os.c

```diff
@@ -350,42 +350,43 @@ os_handle_download_update_rpm_diff (RPMOSTreeOS *interface,
   return TRUE;
 }
 
+static inline void*
+vardict_lookup_ptr (GVariantDict  *dict,
+                    const char    *key,
+                    const char    *fmt)
+{
+  void *val = NULL;
+  g_variant_dict_lookup (dict, key, fmt, &val);
+  return val;
+}
+
+static gboolean
+vardict_lookup_bool (GVariantDict *dict,
+                     const char   *key,
+                     gboolean      dfault)
+{
+  gboolean val = dfault;
+  g_variant_dict_lookup (dict, key, "b", &val);
```

Comment:
One thing that [Coverity](http://www.coverity.com/) does is warn when a function's return value is *sometimes* checked and sometimes not.  So I tend to always check now - it feels like good practice and helps reduce noise in the analyzer.

So this would be:

```
if (g_variant_dict_lookup (dict, key, "b", &val))
  return val;
return default;
```